src/prommis/solid_handling/crusher_solids_properties.py

```diff
@@ -0,0 +1,211 @@
+        self.CaO = Component()
+        self.Fe2O3 = Component()
+
+        self.mw = Param(
+            self.component_list,
+            units=units.kg / units.mol,
+            initialize={
+                "inerts": 60.08e-3,
+                "Sc2O3": (44.946 * 2 + 3 * 15.999) * 1e-3,
+                "Y2O3": (88.905 * 2 + 3 * 15.999) * 1e-3,
+                "La2O3": (138.905 * 2 + 3 * 15.999) * 1e-3,
+                "Ce2O3": (140.116 * 2 + 3 * 15.999) * 1e-3,
+                "Pr2O3": (140.907 * 2 + 3 * 15.999) * 1e-3,
+                "Nd2O3": (144.242 * 2 + 3 * 15.999) * 1e-3,
+                "Sm2O3": (150.36 * 2 + 3 * 15.999) * 1e-3,
+                "Gd2O3": (157.25 * 2 + 3 * 15.999) * 1e-3,
+                "Dy2O3": (162.50 * 2 + 3 * 15.999) * 1e-3,
+                "Al2O3": (26.982 * 2 + 3 * 15.999) * 1e-3,
+                "CaO": (40.078 + 15.999) * 1e-3,
+                "Fe2O3": (55.845 * 2 + 3 * 15.999) * 1e-3,
+            },
+        )
+
+        self.mass_frac_comp_initial = Param(
+            self.component_list,
+            units=units.kg / units.kg,
+            initialize={
+                "inerts": 0.6952,
+                "Al2O3": 0.237,
+                "Fe2O3": 0.0642,
+                "CaO": 3.31e-3,
+                "Sc2O3": 2.77966e-05,
+                "Y2O3": 3.28653e-05,
+                "La2O3": 6.77769e-05,
+                "Ce2O3": 0.000156161,
+                "Pr2O3": 1.71438e-05,
+                "Nd2O3": 6.76618e-05,
+                "Sm2O3": 1.47926e-05,
+                "Gd2O3": 1.0405e-05,
+                "Dy2O3": 7.54827e-06,
+            },
+        )
+
+        self.dens_mass = Param(
+            units=units.kg / units.litre,
+            initialize=2.4,
+            mutable=True,
+        )
+
+        self._state_block_class = CoalRefuseStateBlock
+
+        self.bond_work_index = Param(
+            units=units.W
+            * units.hour
+            / units.kg,  # original unit in equation (kw.hour/tonne)
+            initialize=12,  # Bond work index
+            mutable=True,
+            doc="Bond work index",
+        )
+
+    @classmethod
+    def define_metadata(cls, obj):
+        obj.add_default_units(
+            {
+                "time": units.hour,
+                "length": units.m,
+                "mass": units.kg,
+                "amount": units.mol,
+                "temperature": units.K,
+            }
+        )
+
+
+class _CoalRefuseStateBlock(StateBlock):
+    def fix_initialization_states(self):
+        """
+        Fixes state variables for state blocks.
+
+        Returns:
+            None
+        """
+        # Fix state variables
+        fix_state_vars(self)
+
+        for sbd in self.values():
+            if not sbd.config.defined_state:
+                sbd.sum_mass_frac.deactivate()
+
+
+@declare_process_block_class("CoalRefuseStateBlock", block_class=_CoalRefuseStateBlock)
+class CoalRefuseStateBlockData(StateBlockData):
```

Comment:
We should include a short doc string here. We will likely add a test that will check for this in future, so we should fix this before it becomes an issue.